packages/pigeon/lib/generator_tools.dart

```diff
@@ -624,3 +622,9 @@ class OutputFileOptions<T> {
   /// Options for specified language across all file types.
   T languageOptions;
 }
+
+/// Provides a trailing comma for strings that are greater than 75 len
+String getTrailingComma(String string,
```

Comment:
Since this doesn't always return a comma, the name is confusing. What about `bool needsTrailingComma`, and then the caller can do `needsTrailingComma(...) ? ',' : '';` so it's clear what's happening at the call site?